src/features/classification/security-controls/SecurityControlsItem.js

```diff
@@ -1,49 +1,30 @@
 // @flow
 import * as React from 'react';
-import { FormattedMessage } from 'react-intl';
+import { FormattedMessage, type MessageDescriptor } from 'react-intl';
 
-import type { MessageDescriptor } from 'react-intl';
-
-import Tooltip from '../../../components/tooltip';
-import IconInfo from '../../../icons/general/IconInfo';
-import { bdlBoxBlue } from '../../../styles/variables';
+import { bdlYellorange } from '../../../styles/variables';
+import IconSecurityClassificationSolid from '../../../icons/general/IconSecurityClassificationSolid';
+import { SECURITY_CONTROLS_FORMAT } from '../constants';
+import type { ControlsFormat } from '../flowTypes';
```

Comment:
nit: Similar here. imports should be modules, new line, full path, new line, dot dot path, new line, and then type @jstoffan confirmation please